res/qml/LibraryControlLoadSelectedTrackHandler.qml

```diff
@@ -0,0 +1,38 @@
+import Mixxx 0.1 as Mixxx
+import QtQuick 2.12
+
+/// Usually, this component shouldn't be an (visual) `Item` and use something
+/// like `QtObject` instead. However, for some reason using `QtObject` here
+/// makes Mixxx crash on load (using Qt 5.15.2+kde+r43-1). We can check if this
+/// is fixed upstream once we switch to Qt 6.
+Item {
+    id: root
+
+    property string group // required
+    property bool enabled: true
+
+    signal loadTrackRequested(bool play)
+
+    Mixxx.ControlProxy {
+        group: root.group
+        key: "LoadSelectedTrack"
+        onValueChanged: {
+            if (value == 0 || !root.enabled)
```

Comment:
A `QtObject` does not have an `enabled` property. Receiving keyboard and mouse events is a QtQuick feature and thus managed through the QtQuick qml base type `Item`. This is probably why the `QtObject` version breaks. Maybe because its breaking on evaluating `!undefined` since `root.enabled` is `undefined`)?

Reply:
Also if removing the `Item` dependency still didn't fix it, maybe we're using `QtObject` wrong and the base should be a `Component` instead?

Reply:
`Item` doesn't have an `enabled` property either, I was declared here. And in any case, it should throw a warning, not cause a segfault. `Component` doesn't work either, same issue as with `QtObject`.

Reply:
> It was declared here

Sorry, I missed that, still `Item` does have an enabled property. I guess the declared one shadows the Items one. https://doc.qt.io/qt-5/qml-qtquick-item.html#enabled-prop

> And in any case, it should throw a warning, not cause a segfault.

Yes, of course. The crash is still a bug, though I thought I might have found the issue thats triggering the bug and we could work around it, but I was wrong.